1.0/docs/devguide/styling.md

```diff
@@ -24,92 +40,148 @@ provided via `<style>` tags placed inside the element's local DOM `<template>`.
             display: block;
             border: 1px solid red;
           }
-          #child-element {
+          /* In shady DOM this will become `.child-element.my-element {` */
```

Comment:
We haven't really talked about how shady DOM shims things yet, so it's a little confusing to say, "In shady DOM this becomes ..." This is an implementation detail of how the shady DOM styling works, so if we discuss it, it should be in the context of the shim implementation/shim limitations.